test/wdio/slot-fallback-with-textnode/cmp.test.tsx

```diff
@@ -0,0 +1,19 @@
+import { h } from '@stencil/core';
+import { render, waitForChanges } from '@wdio/browser-runner/stencil';
```

Comment:
The `waitForChanges` helper actually doesn't do much and will be removed in an upcoming version of WDIO.
```suggestion
import { render } from '@wdio/browser-runner/stencil';
```